File: async-scala-client/src/main/scala/io/swagger/client/model/PurchaseInfo.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class PurchaseInfo (
  governmentOrder: Option[String],  // Invoice info for government purchase orders Informação da Nota de Empenho de compras públicas 
orderNumber: Option[String],  // Order info Informação do pedido 
contractNumber: Option[String]  // Agreement info Informação do contrato 
)

File: javascript-closure-angular-client/API/Client/PurchaseInfo.js
goog.provide('API.Client.PurchaseInfo');

/**
 * @record
 */
API.Client.PurchaseInfo = function() {}

/**
 * Invoice info for government purchase orders Informação da Nota de Empenho de compras públicas 
 * @type {!string}
 * @export
 */
API.Client.PurchaseInfo.prototype.governmentOrder;

/**
 * Order info Informação do pedido 
 * @type {!string}
 * @export
 */
API.Client.PurchaseInfo.prototype.orderNumber;

/**
 * Agreement info Informação do contrato 
 * @type {!string}
 * @export
 */
API.Client.PurchaseInfo.prototype.contractNumber;

